client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -155,12 +184,11 @@ const getAddressErrors = (
 	} );
 
 	if ( countriesData[ country ] ) {
-		switch ( country ) {
-			case 'US':
-				if ( ! /^\d{5}(?:-\d{4})?$/.test( postcode ) ) {
-					errors.postcode = translate( 'Invalid ZIP code format' );
-				}
-				break;
+		if (
+			includes( ACCEPTED_USPS_ORIGIN_COUNTRY_CODES, country ) &&
```

Comment:
Are we not going to validate zipcodes for non-US (and PR, etc) countries?

Reply:
Good catch! I think it's better to park this until after the release, it's not trivial and it's also not critical, since it only affects the destination address and that's supposed to be entered at checkout time 99% of the time. Also, address normalization *should* fail if the postcode is invalid, so we need to balance out how much bloat do we want to add vs how important is this. I've created https://github.com/Automattic/wp-calypso/issues/25937 to track this.